__tests__/endpoint.tests/endpoint.test.js

```diff
@@ -156,6 +156,28 @@ describe("articles", () => {
           expect(body.msg).toBe("Please enter a valid query");
         });
     });
+
+    test("GET:200 returns an array of articles with topic being the entered value in topic query", () => {
+      return request(app)
+        .get("/api/articles?topic=mitch")
+        .expect(200)
+        .then(({ body }) => {
+          expect(Array.isArray(body.articles)).toBe(true);
+          expect(body.articles.length).toBeGreaterThan(0);
```

Comment:
Go and check the test data and find out exactly how many there should be

Reply:
@nc-mark-t thanks for the feedback, will update it as soon as I finish the current task